backend/src/event/EVENT_TRANSACTION_RECEIVE.ts

```diff
@@ -0,0 +1,22 @@
+import Decimal from 'decimal.js-light'
+import { User as DbUser } from '@entity/User'
+import { Transaction as DbTransaction } from '@entity/Transaction'
+import { Event as DbEvent } from '@entity/Event'
+import { Event, EventType } from './Event'
+
+export const EVENT_TRANSACTION_RECEIVE = async (
+  user: DbUser,
+  involvedUser: DbUser,
```

Comment:
keep in mind, that both users could be a remote user in context of a x-community-tx.